src/common_swagger_api/schema/webhooks.clj

```diff
@@ -15,7 +15,7 @@
 (defschema WebhookType
   {(optional-key :id)       (describe UUID "A UUID for the type")
    :type                    (describe NonBlankString "Webhook type")
-   (optional-key :template) (describe NonBlankString "Template for this Webhook type")})
+   (optional-key :template) (describe "Template for this Webhook type")})
```

Comment:
This still needs a type to `describe`:
```suggestion
   (optional-key :template) (describe String "Template for this Webhook type")})
```
You can run `lein eastwood` in this repo's directory.
It's a linter that will check for errors without having to build and run this in our actual services first.
If you run it now, it should complain about a syntax error here.

Reply:
The `template` is not used by the UI at the moment. So either allowing blank or not including the key all together makes no difference 👍